src/utils/argument_convert/mod.rs

```diff
@@ -1,195 +0,0 @@
-}
-
-/// Retrieves guild, channel, and message ID from a message URL.
-///
-/// If the URL is malformed, None is returned.
-///
-/// # Examples
-/// ```rust
-/// use serenity::model::prelude::*;
-/// use serenity::utils::parse_message_url;
-///
-/// assert_eq!(
-///     parse_message_url(
-///         "https://discord.com/channels/381880193251409931/381880193700069377/806164913558781963"
-///     ),
-///     Some((
-///         GuildId::new(381880193251409931),
-///         GenericChannelId::new(381880193700069377),
-///         MessageId::new(806164913558781963),
-///     )),
-/// );
-/// assert_eq!(
-///     parse_message_url(
-///         "https://canary.discord.com/channels/381880193251409931/381880193700069377/806164913558781963"
-///     ),
-///     Some((
-///         GuildId::new(381880193251409931),
-///         GenericChannelId::new(381880193700069377),
-///         MessageId::new(806164913558781963),
-///     )),
-/// );
-/// assert_eq!(parse_message_url("https://google.com"), None);
-/// ```
-#[must_use]
-pub fn parse_message_url(s: &str) -> Option<(GuildId, GenericChannelId, MessageId)> {
-    use aformat::{CapStr, aformat};
-
-    for domain in DOMAINS {
-        let prefix = aformat!("https://{}/channels/", CapStr::<MAX_DOMAIN_LEN>(domain));
-        if let Some(parts) = s.strip_prefix(prefix.as_str()) {
-            let mut parts = parts.splitn(3, '/');
-
-            let guild_id = parts.next()?.parse().ok()?;
-            let channel_id = parts.next()?.parse().ok()?;
-            let message_id = parts.next()?.parse().ok()?;
-            return Some((guild_id, channel_id, message_id));
-        }
-    }
-    None
-}
-
-/// Retrieves guild, and channel ID from a channel URL.
-///
-/// If the URL is malformed, None is returned.
-///
-/// # Examples
-/// ```rust
-/// use serenity::model::prelude::*;
-/// use serenity::utils::parse_channel_url;
-///
-/// assert_eq!(
-///     parse_channel_url("https://discord.com/channels/381880193251409931/381880193700069377"),
-///     Some((GuildId::new(381880193251409931), GenericChannelId::new(381880193700069377),)),
-/// );
-/// assert_eq!(
-///     parse_channel_url(
-///         "https://canary.discord.com/channels/381880193251409931/381880193700069377"
-///     ),
-///     Some((GuildId::new(381880193251409931), GenericChannelId::new(381880193700069377),)),
-/// );
-/// assert_eq!(parse_channel_url("https://google.com"), None);
-/// ```
-#[must_use]
-pub fn parse_channel_url(s: &str) -> Option<(GuildId, GenericChannelId)> {
-    use aformat::{CapStr, aformat};
-
-    for domain in DOMAINS {
-        let prefix = aformat!("https://{}/channels/", CapStr::<MAX_DOMAIN_LEN>(domain));
-        if let Some(parts) = s.strip_prefix(prefix.as_str()) {
-            let mut parts = parts.splitn(2, '/');
-
-            let guild_id = parts.next()?.parse().ok()?;
-            let channel_id = parts.next()?.parse().ok()?;
-            return Some((guild_id, channel_id));
-        }
-    }
-    None
-}
```

Comment:
Can you move this to `utils/mod.rs` as well?

Reply:
Done.